runtime/bin/ipfix-rita

```diff
@@ -4,9 +4,12 @@ set -e
 
 # Change dir to script dir
 pushd "$(dirname "$(realpath "${BASH_SOURCE[0]}")")" > /dev/null
-export COMPOSE_FILE="../lib/docker-compose/main.yaml"
+
```

Comment:
Move away from the using the COMPOSE_FILE env variable to set the docker-compose file to use. This way the user can specify override files using `-f`. 

This decision may need to be overturned if we provide a "nicer" main script which doesn't just route arguments directly to docker-compose.